channels/applicationinsights-channel-js/src/TelemetryProcessors/Sample.ts

```diff
@@ -16,8 +16,8 @@ export class Sample implements ISample {
         let _logger = logger || safeGetLogger(null);
         
         if (sampleRate > 100 || sampleRate < 0) {
-            _logger.throwInternal(LoggingSeverity.WARNING,
-                _InternalMessageId.SampleRateOutOfRange,
+            _logger.throwInternal(eLoggingSeverity.WARNING,
+                _eInternalMessageId.SampleRateOutOfRange,
```

Comment:
Changing all references from the full name to the const enum which is replaced with the numeric value during compile.